### PR TITLE
Prepare merge into official repository

### PR DESCRIPTION
**Pull request by athieriot**
Following our discussion on Discuss, here the code of drone-artifactory

### CODE REVIEW

**Comment by bradrydzewski** [comment]
cc @nlf would you mind a quick code review for our first node plugin? :smile:

**Comment by nlf** [comment]
LGTM. nice work @athieriot

**Comment by athieriot** [comment]
Thank you!

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+build:
+  image: node:4.2
+  commands:
+    - npm install
+
+publish:
+  docker:
+    username: athieriot
```
Comment (tboerger) [inline .drone.yml]: `$$DOCKER_USER`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN apk update && \
+    apk add nodejs && \
+    rm -rf /var/cache/apk/*
+
+WORKDIR /node
+
+COPY package.json /node/
+RUN npm install
+COPY . /node/
```
Comment (tboerger) [inline Dockerfile]: Why do you copy everything? Should be enough to add `index.js`, right?

Comment (athieriot) [inline Dockerfile]: Well... yes.
Didn't thought about that. 

Need to update the other plugins as well

**Comment by tboerger** [comment]
LGTM

**Pull request merged**